shortcircuit-api/src/main/java/com/powsybl/shortcircuit/InitialVoltageProfile.java

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.shortcircuit;
+
+/**
+ * @author Coline Piloquet <coline.piloquet at rte-france.com>
+ */
+public enum InitialVoltageProfile {
```

Comment:
I think it is more a "mode", as we do in open load flow parameters. So it could be `InitialVoltageProfileMode`.